addons/mail/static/src/core/common/message.scss

```diff
@@ -44,16 +44,21 @@
 .o-mail-Message-body {
     font-family: "text-emoji", $font-family-base;
 
-    & > p {
+    p:last-child {
```

Comment:
![image](https://github.com/user-attachments/assets/ab245e78-6fba-47e3-96a7-5bbbd350eaea)
lists generate unwanted extra margin at the end too